qurro/q2/_type.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018--, Qurro development team.
+#
+# NOTE: This is based on songbird's songbird/q2/_stats.py file.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from q2_types.sample_data import SampleData
+from qiime2.plugin import model, SemanticType
+
+QarcoalLogRatios = SemanticType(
```

Comment:
It's ultimately your call, but I think naming this something like `SampleData[LogRatios]` instead of `SampleData[QarcoalLogRatios]` might be a cleaner option -- especially if other programs come around that can accept these types as inputs, etc.

Reply:
Definitely makes sense. `SampleData[LogRatios]` feels cleaner and more extensible.